packs/rabbitmq/templates/rabbit.nomad.tpl

```diff
@@ -0,0 +1,107 @@
+job [[ template "job_name" . ]] {
+
+  datacenters = [[ .rabbitmq.datacenters | toJson ]]
+  type        = "service"
+
+  constraint {
+    attribute = "${attr.consul.version}"
+    operator  = "is_set"
+  }
+
+  group "cluster" {
+    count = [[ .rabbitmq.cluster_size ]]
+
+    update {
+      max_parallel = 1
+    }
+
+    migrate {
+      max_parallel     = 1
+      health_check     = "checks"
+      min_healthy_time = "5s"
+      healthy_deadline = "30s"
+    }
+
+    network {
+      port "amqp" {
+        to = 5671
+        [[- template "port" .rabbitmq.port_amqp ]]
+      }
+      port "ui" {
+        to     = 15671
+        [[- template "port" .rabbitmq.port_ui ]]
+      }
+      port "discovery" {
+        to = 4369
+        static = 4369
```

Comment:
Do discovery and clustering have to be at these static ports?

No big deal if so, but might be nice to make these parameterized.

Reply:
I think I can make it configurable, I'll need to check about the port configuration first though.

Reply:
Done; and noted in readme and description that they must be static assigned ports